src/components/SpruceForm/Widgets/LeafyGreenWidgets.tsx

```diff
@@ -233,11 +233,13 @@ export const LeafyGreenRadio: React.VFC<EnumSpruceWidgetProps> = ({
       >
         {enumOptions.map((o) => {
           const optionDisabled = enumDisabled?.includes(o.value) ?? false;
+          const { description } = o.schema ?? {};
           return (
             <Radio
               key={o.value}
               value={o.value}
               disabled={disabled || optionDisabled}
+              description={description}
```

Comment:
Would love to see some tests in `Spruceform.test.tsx`I don't think we have any for radio buttons.  :)